src/views/landing/ExperiencedPlayerHome/components/BannerSlide.vue

```diff
@@ -33,17 +36,65 @@
 
     @Prop({}) carousel_subtitle!: string;
 
+    @Prop() designs_to_be_synthesized!: number;
+
+    @Prop() max_designs!: number;
+
+    @Prop() total_submitted_solutions_of_user!: number;
+
+    @Prop() total_submitted_solutions!: number;
+
     @Prop({}) nid!: number;
 
     get heroImage() {
       return this.banner_image || DefaultHero;
     }
+
+    progressCircles = [
+      {
+        name: 'progress-circle:designs-submissions',
+        progress: this.designs_to_be_synthesized,
+        total: this.max_designs,
+      },
+      {
+        name: 'progress-circle:my-submissions',
+        progress: this.total_submitted_solutions_of_user,
+        total: this.total_submitted_solutions,
+      },
+    ];
```

Comment:
@luxaritas thanks made these changes for the first two bullets. I will ask Tamas for an example for what a not null example for the `project_closes` field looks like to make the countdown timer and also to add the data to the endpoints so this can show in the lab

Reply:
added countdown using timestamp
![image](https://user-images.githubusercontent.com/45175645/82896543-6e9ce000-9f56-11ea-8412-fea38f0afa5b.png)